#actual speed may be lower due to track curves or so, but never higher!
float32 max_speed
#standard=0, switch_left=1, switch_right=2, turn_left=3, turn_right=4, straight_forward=5
int32 control_metadata
